http-test/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set +e
+while
+(exec 6<>/dev/tcp/localhost/8080) 2>/dev/null
+[ $? == 1 ]
+do echo Waiting for Go Drink; sleep 1; done
+set -e
+
```

Comment:
Since the scripts expects to be executed from the repository root, and fails if it is executed e.g. directly from the http-test dir, I think it would make sense to warn a user explicitly about this easy-to-make mistake. Or, even better, navigate to the repository root right here in the script.

Reply:
Yeah... It might be even better to allow passing in the input/ output directory... I'll take a look.

Reply:
I fixed it by reworking the script to accept a bunch of parameters and aborting if the env file isn't found.

Reply:
Nice, that's a definitely a clean solution! I run shellcheck and quoted some vars, hopefully this will prevent some bugs when people run this from unholy locations...

See https://github.com/koalaman/shellcheck/wiki/SC2086 for more information on this.